tests/chainer_tests/functions_tests/rnn_tests/test_function_n_step_gru.py

```diff
@@ -1,91 +1,121 @@
-import unittest
-
 import numpy
 
 import chainer
-from chainer.backends import cuda
-from chainer import functions
-from chainer import gradient_check
+import chainer.functions as F
 from chainer import testing
-from chainer.testing import attr
+from chainer.testing import backend
 
 
 def sigmoid(x):
```

Comment:
Are you using this function?

Reply:
https://github.com/dido1998/chainer/blob/b2b8cbcd8fc34e6ca3af9f631982d638a3c2128f/tests/chainer_tests/functions_tests/rnn_tests/test_function_n_step_gru.py#L122 it is being used here

Reply:
cool, i tried to search it but i probably typed the name wrong, thanks!